hugo/content/Upgrade/upgrade35to4.md

```diff
@@ -64,12 +66,14 @@ To verify cluster status, run
 	pgo test <clustername> -n <namespace>
 Output should be similar to:
 ```
-psql -p 5432 -h 10.104.74.189 -U postgres postgres is Working
-psql -p 5432 -h 10.104.74.189 -U postgres userdb is Working
-psql -p 5432 -h 10.104.74.189 -U primaryuser postgres is Working
-psql -p 5432 -h 10.104.74.189 -U primaryuser userdb is Working
-psql -p 5432 -h 10.104.74.189 -U testuser postgres is Working
-psql -p 5432 -h 10.104.74.189 -U testuser userdb is Working
+cluster : mycluster
+        Services
+                primary (10.106.70.238:5432): UP
+        Instances
+                primary (mycluster-7d49d98665-7zxzd): UP
 ```
 ##### Step 7
 Scale up to the required number of replicas, as needed.
+
+It is also recommended to take full backups of each pgcluster once the upgrade is completed due to versio
+n differences between the old and new clusters.
```

Comment:
I see an extra break in the middle of `version` here.

Reply:
Yep, corrected.